app/models/assignment.rb

```diff
@@ -22,6 +22,7 @@ class Assignment < ActiveRecord::Base
 
   validates :title, presence: true
   validates :title, length: { maximum: 60 }
+  validates :title, exclusion: { in: AssignmentsController.action_methods.to_a }
```

Comment:
I'm not a fan of this method of reserving worlds. If we want to make reserved words, there should be a list that we can manage and update.